### ISSUE TITLE
The Dapr operator container is in crashloop backoff in Helm chart 1.11.1

### ISSUE THREAD

**Issue by polinchw** [dapr/dapr#6655]
## Expected Behavior

The Dapr operator container should be stable.

## Actual Behavior

The Dapr operator pod is in crash loop back off after installing the Dapr 1.11.1 Helm chart.

This is what is observed in the logs:

`time="2023-07-11T17:22:49.937549597Z" level=fatal msg="error running operator: error running operator: failed to wait for rollout caches to sync: timed out waiting for cache to be synced" instance=dapr-operator-7cd6d96dfd-9wbj7 scop │
│ Stream closed EOF for dapr-system/dapr-operator-7cd6d96dfd-9wbj7 (dapr-operator)  `

## Steps to Reproduce the Problem

Install this helm chart:

dapr/dapr    1.11.1

## Release Note

RELEASE NOTE:

**Comment by ItalyPaleAle** [comment]
Moved to dapr/dapr since this seems to be an issue with the operator and not a component

**Comment by polinchw** [comment]
> Can you paste the entire log from the operator?

`time="2023-07-20T19:39:19.093290259Z" level=info msg="log level set to: info" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:19.093488958Z" level=info msg="metrics server started on :9090/" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.metrics type=log ver=1.11.1
time="2023-07-20T19:39:19.093750557Z" level=info msg="starting Dapr Operator -- version 1.11.1 -- commit 204163e75fcfd4d56b6cdbb849265b223372aab0" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
I0720 19:39:20.145552       1 request.go:690] Waited for 1.014473633s due to client-side throttling, not priority and fairness, request: GET:https://10.0.0.1:443/apis/traefik.containo.us/v1alpha1?timeout=32s
time="2023-07-20T19:39:21.806512752Z" level=info msg="Dapr Operator is starting" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
I0720 19:39:21.906907       1 leaderelection.go:248] attempting to acquire leader lease dapr-system/operator.dapr.io...
time="2023-07-20T19:39:21.906809089Z" level=info msg="Healthz server is listening on :8080" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:22.008315622Z" level=info msg="Getting TLS certificates" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:22.008388022Z" level=info msg="Starting watch for certs on filesystem: /var/run/dapr/credentials" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:22.008441122Z" level=info msg="TLS certificates loaded successfully" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:22.047069082Z" level=info msg="Conversion webhook for \"subscriptions.dapr.io\" is up to date" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:22.047096182Z" level=info msg="Starting api server" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:22.047108482Z" level=info msg="starting gRPC server on port 6500" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator.api type=log ver=1.11.1
time="2023-07-20T19:39:22.047492681Z" level=info msg="Dapr Operator started" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
I0720 19:39:39.602534       1 leaderelection.go:258] successfully acquired lease dapr-system/operator.dapr.io
time="2023-07-20T19:39:39.602670977Z" level=info msg="DaprWatchdog worker starting" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:39.602782576Z" level=info msg="DaprWatchdog worker started" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:39.602806176Z" level=info msg="DaprWatchdog started checking pods" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:39:39.904740696Z" level=info msg="DaprWatchdog completed checking pods" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:41:39.603980268Z" level=info msg="Healthz server is shutting down" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:41:39.604213867Z" level=info msg="DaprWatchdog worker stopping" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:41:39.604261667Z" level=info msg="DaprWatchdog worker stopped" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
time="2023-07-20T19:41:39.604737465Z" level=fatal msg="error running operator: error running operator: failed to wait for rollout caches to sync: timed out waiting for cache to be synced" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1
Stream closed EOF for dapr-system/dapr-operator-7cd6d96dfd-szqvn (dapr-operator)
`

**Comment by lynn-orrell** [comment]
Reformatted for readability...

```
time="2023-07-20T19:39:19.093290259Z" level=info msg="log level set to: info" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:19.093488958Z" level=info msg="metrics server started on :9090/" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.metrics type=log ver=1.11.1 
time="2023-07-20T19:39:19.093750557Z" level=info msg="starting Dapr Operator -- version 1.11.1 -- commit 204163e75fcfd4d56b6cdbb849265b223372aab0" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 I0720 19:39:20.145552       1 request.go:690] Waited for 1.014473633s due to client-side throttling, not priority and fairness, request: GET:https://10.0.0.1:443/apis/traefik.containo.us/v1alpha1?timeout=32s 
time="2023-07-20T19:39:21.806512752Z" level=info msg="Dapr Operator is starting" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 I0720 19:39:21.906907       1 leaderelection.go:248] attempting to acquire leader lease dapr-system/operator.dapr.io... 
time="2023-07-20T19:39:21.906809089Z" level=info msg="Healthz server is listening on :8080" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:22.008315622Z" level=info msg="Getting TLS certificates" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:22.008388022Z" level=info msg="Starting watch for certs on filesystem: /var/run/dapr/credentials" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:22.008441122Z" level=info msg="TLS certificates loaded successfully" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:22.047069082Z" level=info msg="Conversion webhook for \"subscriptions.dapr.io\" is up to date" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:22.047096182Z" level=info msg="Starting api server" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:22.047108482Z" level=info msg="starting gRPC server on port 6500" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator.api type=log ver=1.11.1 
time="2023-07-20T19:39:22.047492681Z" level=info msg="Dapr Operator started" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 I0720 19:39:39.602534       1 leaderelection.go:258] successfully acquired lease dapr-system/operator.dapr.io 
time="2023-07-20T19:39:39.602670977Z" level=info msg="DaprWatchdog worker starting" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:39.602782576Z" level=info msg="DaprWatchdog worker started" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:39.602806176Z" level=info msg="DaprWatchdog started checking pods" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:39:39.904740696Z" level=info msg="DaprWatchdog completed checking pods" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:41:39.603980268Z" level=info msg="Healthz server is shutting down" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:41:39.604213867Z" level=info msg="DaprWatchdog worker stopping" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:41:39.604261667Z" level=info msg="DaprWatchdog worker stopped" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 
time="2023-07-20T19:41:39.604737465Z" level=fatal msg="error running operator: error running operator: failed to wait for rollout caches to sync: timed out waiting for cache to be synced" instance=dapr-operator-7cd6d96dfd-szqvn scope=dapr.operator type=log ver=1.11.1 Stream closed EOF for dapr-system/dapr-operator-7cd6d96dfd-szqvn (dapr-operator) 
```
